swarm/api/http/server.go

```diff
@@ -591,6 +591,12 @@ func (s *Server) HandleGetFile(w http.ResponseWriter, r *Request) {
 func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	s.logDebug("HTTP %s request URL: '%s', Host: '%s', Path: '%s', Referer: '%s', Accept: '%s'", r.Method, r.RequestURI, r.URL.Host, r.URL.Path, r.Referer(), r.Header.Get("Accept"))
 
+	if r.RequestURI == "/" && strings.Contains(r.Header.Get("Accept"), "application/json") {
+		w.Header().Set("Content-Type","application/json")
+		w.WriteHeader(http.StatusOK)
+		return
```

Comment:
It should actually write valid JSON in the response. The simplest thing is to just write a string:

```go
json.NewEncoder(w).Write("welcome to Swarm!")
```

Reply:
@lmars Gotcha, replace all the current response code with just `json.NewEncoder`?

Reply:
@cooganb The response code can stay, but you have to add JSON content to the response